onnxruntime/contrib_ops/cuda/bert/attention.cc

```diff
@@ -42,11 +42,11 @@ Status Attention<T>::ComputeInternal(OpKernelContext* context) const {
   ORT_RETURN_IF_ERROR(CheckInputs(input->Shape(), weights->Shape(), bias->Shape(), mask_index, past));
 
   // Input and output shapes:
-  //   Input 0 - input       : (batch_size, sequence_length, hidden_size)
+  //   Input 0 - input       : (batch_size, sequence_length, hidden_size) or (sequence_length, batch_size, hidden_size)
   //   Output 0 - output     : (batch_size, sequence_length, hidden_size)
```

Comment:
Just wondering. output is not swapped when input is swapped?

Reply:
Good catch! the output should have the same shape as input. let me update the comment